inc/database.inc.php

```diff
@@ -67,7 +67,6 @@ function dbConnect() {
     global $db_charset;
     global $db_file;
     global $db_debug;
-    global $db_ssl_ca;
```

Comment:
It is strange that it's not used anymore, probably someone removed it previously, doh...